llvm/include/llvm/LTO/LTO.h

```diff
@@ -197,10 +231,30 @@ class InputFile {
 /// A ThinBackend defines what happens after the thin-link phase during ThinLTO.
```

Comment:
Can you update this and have a separate comment for the renamed function type alias and the new struct type?